Scripts/Python/step_response_identification.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
+import numpy as np
+from typing import Tuple
+import control
+from control.matlab import step
+import matplotlib.pyplot as plt
+import easygui
+
+def check_data(dataframe):
+    """
+    Check if the DataFrame represents a step response.
+
+    Args:
+        dataframe (pd.DataFrame): The DataFrame to check.
+
+    """
+
+    assert 'H1_ADD_NOISE' in dataframe.columns, "Invalid experiment CSV file"
+
+    dataframe_test = dataframe.copy()
+    dataframe_test.drop_duplicates(subset='H1_ADD_NOISE', keep='first', inplace=True)
+    nonzero_points = dataframe_test[dataframe_test['H1_ADD_NOISE'] != 0]
+    assert len(nonzero_points) == 1, "The experiment is not a step response"
+
+
+def load_data(file_path: str) -> pd.DataFrame:
+    """
+    Load data from a CSV file.
+
+    Args:
+        file_path (str): The path to the CSV file.
+
+    Returns:
+        pd.DataFrame: The loaded data as a pandas DataFrame.
+    """
+    dataframe = pd.read_csv(file_path)
+    check_data(dataframe)
+    dataframe.set_index('TIME', inplace=True)
+    starting_time = dataframe.loc[dataframe['H1_ADD_NOISE'] > 0.0].index.values[0]
+    dataframe = dataframe.loc[starting_time:]
+    dataframe.index = (dataframe.index - starting_time) / 1000
+    return dataframe
+
+def calculate_parameters(dataframe: pd.DataFrame) -> Tuple[float, float, float, float, float]:
```

Comment:
Please, use "identification" as a name for this function. This will be aligned with future standards.